.travis.yml

```diff
@@ -2,8 +2,9 @@ language: python
 addons:
```

Comment:
add 
```
dist: xenial
```
or it will use ubuntu 14 or so...

Reply:
done